where-to-travel/src/main/webapp/index.html

```diff
@@ -1,28 +1,155 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
```

Comment:
Does this need a closing '/'? We don't do that with the links or images. I believe preferred style is to not add '/' to tags that do not have a separate closing tag. [Google style](https://google.github.io/styleguide/htmlcssguide.html#General_Meta_Rules) doesn't close it when they reference it.

Reply:
fixed!